test/tests/interfaces/pbs_sched_interface_test.py

```diff
@@ -41,22 +41,32 @@
 
 @tags('multisched')
 class TestSchedulerInterface(TestInterfaces):
+
     """
     Test suite to test different scheduler interfaces
     """
 
     def setUp(self):
         # delete all schedulers except default
-        self.server.manager(MGR_CMD_DELETE, SCHED, id="@default")
+        self.server.manager(MGR_CMD_DELETE,
+                            SCHED,
+                            id="@default")
+        self.server.manager(MGR_CMD_CREATE,
+                            SCHED,
+                            id="TestCommonSched")
         TestInterfaces.setUp(self)
 
     def test_create_successful_scheduler_object(self):
         """
         Creates a scheduler object succcessfully .
         """
-        a = {'sched_cycle_length': 12000}
-        rc = self.server.manager(MGR_CMD_CREATE, SCHED, a, id="test1")
-        self.assertTrue(rc == 0)
+        try:
+            self.server.manager(MGR_CMD_CREATE,
+                                SCHED,
+                                {'sched_cycle_length': 12000},
+                                id="testCreateSched")
+        except PbsManagerError, e:
+            self.fail(e.msg)
```

Comment:
These test cases are end user's use case driven( 'how usecase are executed by end user'). I consider successful completion of operation is also one test case. Hence there are two test cases on CREATE and DELETE. 
I don't  have any reservation on this.  I am OK to remove these test cases.  

At the end of of test suite execution, framework prints 
" run: 1, succeeded: 1, failed: 0, errors: 0, skipped: 0, timedout: 0".
If the exception is not caught then it is shown as Error, if otherwise it is shown as Fail.  

That explains what i would like to convey from testcases. It helps me in differentiating error test cases. Again i am ok to remove them, if it is acceptable to treat Error and Failed test cases as same.